lighthouse-core/test/runner.js

```diff
@@ -77,67 +67,114 @@ describe('Runner', () => {
     });
   });
 
-  it('throws for unknown gatherers', () => {
+  it('throws when given neither passes nor artifacts', () => {
     const url = 'https://example.com';
     const flags = {
       auditWhitelist: null
     };
-    const config = {
-      passes: [{
-        gatherers: ['fuzz']
-      }],
+    const config = new Config({
       audits: [
         'is-on-https'
       ]
-    };
+    }, flags.auditWhitelist);
 
     return assert.throws(_ => Runner.run(fakeDriver, {url, config, flags}),
-        /Unable to locate/);
+        /The config must provide passes/);
   });
 
-  it('throws when given neither passes nor artifacts', () => {
+  it('accepts existing artifacts', () => {
     const url = 'https://example.com';
     const flags = {
       auditWhitelist: null
     };
-    const config = {
+    const config = new Config({
       audits: [
         'is-on-https'
-      ]
+      ],
+
+      artifacts: {
+        HTTPS: true
+      }
+    }, flags.auditWhitelist);
+
+    return assert.doesNotThrow(_ => Runner.run(fakeDriver, {url, config, flags}));
```

Comment:
should maybe use `{}` instead of `fakeDriver` here because when we're running in artifact-only mode it should never make calls into the driver (an instance of chrome probably isn't even available)